### ISSUE TITLE
Currency Fields type number in Forms - no comma?

### ISSUE THREAD

**Issue by Catzenjaeger** [PHPFusion/PHPFusion#1032]
![unbenannt](https://cloud.githubusercontent.com/assets/19621858/18246937/613376ca-736f-11e6-8c55-1a2d15e01eb9.JPG)

```
                // Formfield "Figure Usedprice"
                echo form_text("figure_usedprice", $locale['figurelib/admin/figurelib.php_052'], $data['figure_usedprice'], [
                    "placeholder"   => $locale['figurelib/admin/figurelib.php_053'],
                    "width"         => "100%",
                    "inline"        => true,
                    "min"           => 0,
                    "prepend_value" => "&#036;",
                    "type"          => "number"
                ]);
```

**Comment by meangczac** [comment]
I will make the price validation later.

**Comment by meangczac** [comment]
The price validation for this will be on BIGINT(20) format so if you're going to use price have it do like that.

If 10,00 ... store as 100000  and then callback is divide by 1000. Only then round to 2.

This method is the most precise when addding/subtracting/dividing money.

After that to display, the field need to show callback as:

```
$number = 1234.56;
setlocale(LC_MONETARY,"en_US");
echo money_format("The price is %i", $number);

http://www.w3schools.com/php/func_string_money_format.asp
```

As setlocale is done at core which 9.02 not support yet, that is why Dynamics did not follow up on the development. 

Again, this feature will only be available at 9.1 above. I put the statement here so infusion makers know what to expect.
